x-pack/plugins/ml/common/types/modules.ts

```diff
@@ -90,8 +90,14 @@ export interface DataRecognizerConfigResponse {
   };
 }
 
-export type GeneralOverride = any;
-
 export type JobOverride = Partial<Job>;
+export type GeneralJobsOverride = Omit<JobOverride, 'job_id'>;
+export type JobSpecificOverride = JobOverride & { job_id: Job['job_id'] };
+
+export function isGeneralJobOverride(override: JobOverride): override is GeneralJobsOverride {
+  return override.job_id === undefined;
+}
+
+export type GeneralDatafeedsOverride = Partial<Omit<Datafeed, 'job_id' | 'datafeed_id'>>;
 
-export type DatafeedOverride = Partial<Datafeed>;
+export type DatafeedOverride = Partial<Datafeed> & { job_id: Job['job_id'] };
```

Comment:
i missed the on the last review, to override a datafeed, you can use either a `job_id` or `datafeed_id`

Reply:
changed it to `Partial<Datafeed>` in https://github.com/elastic/kibana/pull/60656/commits/085960a936a2882f011597b75bc88831d504de73